config/common/webhook/clusterrole-webhook.yaml

```diff
@@ -40,4 +40,19 @@ rules:
       - list
       - watch
       - update
-
+  - apiGroups:
+      - ""
+      - "apps"
+      - "batch"
+      - "apps.openshift.io"
```

Comment:
please add the openshift specific rules via patch just when installing for openshift: `apps.openshift.io` and `deploymentconfigs`